src/doc/src/commands/cargo-test.md

```diff
@@ -386,7 +386,7 @@ the &quot;short&quot; rendering from rustc. Cannot be used with <code>human</cod
 contains embedded ANSI color codes for respecting rustc's default color
 scheme. Cannot be used with <code>human</code> or <code>short</code>.</li>
 <li><code>json-render-diagnostics</code>: Instruct Cargo to not include rustc diagnostics in
-JSON messages printed, but instead Cargo itself should render the
+in JSON messages printed, but instead Cargo itself should render the
```

Comment:
running the tool removed your edit, so you were editing one of the code gen files rather than source. I'm on mobile, so can't point you to what file to edit but hopefully you can find it.

Reply:
FYI, must be this file https://github.com/rust-lang/cargo/blob/master/src/doc/man/includes/options-message-format.md

Reply:
🤦 my bad! Adjusting this now